Feature/reports timezone resolver (#825)
livechat/livechat-public-docs: content/data-reporting/reports-api/v3.3/index.mdx

#### Request

| Query String          | Required | Notes                                                                                                                                                                                                     |
| --------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `to`                  | yes      | Date in the RFC3339 format. If `timezone` provided, the Timezone from this format will be ignored.                                                                                                        |

Reviewer: ```suggestion
| `to`                  | yes      | Date in the RFC3339 format, which also contains a timezone. This timezone will be used if no `timezone` is provided. |
```

Reviewer: I rephrased the sentence a little. WDYT? It's easier to understand, IMO.

Author: It's nice, thanks!

> | `from`                | yes      | Date in the RFC3339 format. If `timezone` provided, the Timezone from this format will be ignored.                                                                                                        |

Reviewer: Same as above

| `agents`              | no       | Agent emails separated by a comma; if not specified, returns the data for all agents within the license.        |
| `groups`              | no       | Group IDs separated by a comma                                                                                  |
| `tags`                | no       | Names of tags separated by a comma                                                                              |
| `customer_client_ids` | no       | Client IDs separated by a comma                                                                                 |
| `distribution`        | no       | Possible values: `hour`, `day-hours`, `day`, `month`; defaults to `day`                                         |

> | `timezone`            | no       | Timezone in the [TZ format](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. America/Phoenix) |

Reviewer 2: please mention the default value

> | `timezone`            | no       | Timezone in the [TZ format](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. America/Phoenix). Defaults to requester Timezone. If not present, `from` is parsed to get it's Timezone.   |

Reviewer: > If not present, `from` is parsed to get **its** timezone.  

"its" suggest the requester is not a person, and I believe it's okay if we assume an application is the requester. But to avoid confusion, I suggest replacing "its" with the phrase "the requester's timezone".

#### Request

| Query String          | Required | Notes                                                                                                                                                                                                     |
| --------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `to`                  | yes      | Date in the RFC3339 format. If `timezone` provided, the Timezone from this format will be ignored.                                                                                                        |

Reviewer: Same as above.

> | `from`                | yes      | Date in the RFC3339 format. If `timezone` provided, the Timezone from this format will be ignored.                                                                                                        |

Reviewer: Same as above.

| `distribution`        | yes      | Possible values: `hour`, `day-hours`, `day`, `month`, `year`                                                                                                                                              |

> | `timezone`            | no       | Timezone in the [TZ format](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. America/Phoenix). Defaults to requester Timezone. If not present, `from` is parsed to get it's Timezone.   |

Reviewer: Same as above.
